lib/good_job/lockable.rb

```diff
@@ -13,7 +13,7 @@ module Lockable
 
         query = cte_table.project(cte_table[:id])
                   .with(composed_cte)
-                  .where(Arel.sql(sanitize_sql_for_conditions(["pg_try_advisory_lock(('x'||substr(md5(:table_name || \"#{cte_table.name}\".\"#{primary_key}\"::text), 1, 16))::bit(64)::bigint)", { table_name: table_name }])))
+                  .where(Arel.sql(sanitize_sql_for_conditions(["pg_try_advisory_lock(('x'||substr(md5(:table_name || \"#{cte_table.name}\".\"#{primary_key}\"::text), 1, 16))::bit(64)::bigint)", { table_name: quoted_table_name }])))
```

Comment:
I think you have the logic reversed on these. 

The parameterized value does not need to be quoted because it is intended to be inserted into the query as a parameterized string. 

The interpolated table/columns are the ones that need to be quoted. (Sorry, I'm on my phone so I can't write out code easily). 

Eg

```ruby
.where(Arel.sql(sanitize_sql_for_conditions(["pg_try_advisory_lock(('x'||substr(md5(:table_name || #{quoted_table_name(cte_table.name)}.#{quoted_column_name(primary_key)}::text), 1, 16))::bit(64)::bigint)", { table_name: table_name }])))
```

Reply:
I reverted this change as it was unrelated. But than you for mentioning the quotation thing. I removed extra quotations now. The only change I made is in joins_advisory_lock scope which adds quoted_table_name and quoted_primary_key